openapi/kas-fleet-manager.yaml

```diff
@@ -892,6 +892,8 @@ components:
               type: boolean
```

Comment:
We should also bump the version https://github.com/bf2fc6cc711aee1a0c2a/kas-fleet-manager/blob/a8cbfdd30d7c48eceb33f68a625e4980577aaa08/openapi/kas-fleet-manager.yaml#L5 to 1.4.0